apps/server/src/modules/provisioning/dto/provisioning-user-output.dto.ts

```diff
@@ -11,17 +11,20 @@ export class ProvisioningUserOutputDto {
 		this.lastName = user.lastName;
 		this.roleNames = user.roleNames;
 		this.schoolId = user.schoolId;
+		this.externalId = user.externalId;
 	}
 
 	id?: EntityId;
 
-	email: string;
+	email?: string;
```

Comment:
email should probably not be optional. the strategy should be responsible for creating a valid email field.

Reply:
fixed